test/unit/inbound/save-schedule.test.js

```diff
@@ -0,0 +1,34 @@
+const db = require('../../../app/data')
+
+const saveSchedule = require('../../../app/inbound/return/save-schedule')
+
+let settlement
+let savedSettlement

```

Comment:
this doesn't need to be a `let`

we can define this in the test which uses it on line 22 as `const`